tests/ImageSharp.Benchmarks/Codecs/Qoi/IdentifyQoi.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using BenchmarkDotNet.Attributes;
+using SixLabors.ImageSharp.Formats;
+using SixLabors.ImageSharp.Formats.Qoi;
+using SixLabors.ImageSharp.Tests;
+
+namespace SixLabors.ImageSharp.Benchmarks.Codecs.Qoi;
+
+[Config(typeof(Config.ShortMultiFramework))]
+public class IdentifyQoi
+{
+    private byte[] qoiBytes;
+
+    private string TestImageFullPath => Path.Combine(TestEnvironment.InputImagesDirectoryFullPath, this.TestImage);
+
+    [Params(TestImages.Qoi.TestCardRGBA, TestImages.Qoi.TestCard, TestImages.Qoi.QoiLogo, TestImages.Qoi.EdgeCase, TestImages.Png.Bike)]
+    public string TestImage { get; set; }
+
+    [GlobalSetup]
+    public void ReadImages() => this.qoiBytes ??= File.ReadAllBytes(this.TestImageFullPath);
+
+    [Benchmark]
+    public ImageInfo Identify()
```

Comment:
This benchmark doesn't give us much info, since there is no baseline to compare to, and IMHO benchmarking Identify is not really valuable in general.

The only benchmark I would find really interesting here is one that compares decoding a larger QOI to the decoding of a PNG (by the system decoder and the ImageSharp PNG decoder). I wouldn't bother with the rest.